ai/coding-assistant.md

```diff
@@ -0,0 +1,70 @@
+
+## Installation
+
+To install the Telerik Blazor Copilot extension:
+
+1. Go the [TelerikBlazor GitHub App](https://github.com/apps/telerikblazor) page and click the **Install** button.
+1. You will see a list that includes your GitHub account and all GitHub organizations that you are part of. Normally, select your GitHub account.
+1. Click the **Install &amp; Authorize** button. This will authorize the GitHub Copilot extension to integrate with your GitHub account.
+1. Enter your GitHub password.
+1. You will be redirected to telerik.com. Enter your Telerik account credentials if prompted. This will authorize the GitHub Copilot extension to integrate with your Telerik account.
+1. Upon successful Telerik authentication, you will be redirected once again to a page that confirms successful Copilot extension installation.
+1. Restart your [Copilot-enabled apps](https://docs.github.com/en/copilot/building-copilot-extensions/about-building-copilot-extensions#supported-clients-and-ides) (for example, Visual Studio and VS Code).
+1. Start a new chat session in Copilot.
+
+## Usage
+
+To use the Telerik Blazor Copilot extension:
+
+1. Open the GitHub Copilot chat window in your [Copilot-enabled app](https://docs.github.com/en/copilot/building-copilot-extensions/about-building-copilot-extensions#supported-clients-and-ides) (for example, Visual Studio or VS Code).
+1. Make sure you are in **Ask** mode and not in **Edit** or **Agent** mode. The Edit and Agent mode do not use the Telerik Copilot extension. However, the Agent mode can use the [Telerik Blazor MCP server](slug:ai-mcp-server).
+1. Start your prompt with `@telerikblazor` and type your request. Make sure that `@telerikblazor` is recognized and highlighted, otherwise the extension may not be installed.
+1. Verify that you see **TelerikBlazor working...** in the output.
+1. Grant permission to the Telerik Blazor extension to read your workspace files.
+1. If you want to prompt for information or code that are not related to your previous prompts, it is a good practice to start a new session in a new chat window, so that the context is not polluted by irrelevant old information.
+
+### Sample Prompts
+
+The following list describes how your prompts may look like:
+
+* `@telerikblazor` Generate a Grid with sorting and paging enabled. Bind the Grid to a Person model and provide dummy data.
+* `@telerikblazor` What is DebounceDelay for the Telerik TextBox?
+* `@telerikblazor` Why am I getting a TelerikBlazor was undefined JavaScript error?
+
+## Number of Requests
+
+> The usage limits below will take place after after June 28, 2025. Until then, each customer with an active commercial or trial license can make up to 300 daily requests.
+
+The Telerik Blazor Copilot extension allows the following maximum number of requests, depending on your license type:
+
+* Perpetual licenses: 50 requests per year
```

Comment:
You may want to link the licensing FAQ that explains the license types: https://www.telerik.com/purchase/faq/licensing-purchasing